lib/PrimitiveHelpers.js

```diff
@@ -44,6 +45,30 @@ function primitivesShareAttributeAccessor(primitive, comparePrimitive) {
 }
 
 /**
+ * Compare two primitives to check if they share an transformable attribute (position and normal) accessor.
+ *
+ * @param {Object} primitive The first primitive to compare.
+ * @param {Object} comparePrimitive The second primitive to compare.
+ * @returns {Boolean} True if primitives share an attribute, false otherwise.
+ */
+function primitivesShareTransformableAttributeAccessor(primitive, comparePrimitive) {
```

Comment:
Maybe add an extra optional argument to `primitivesShareAttributeAccessor` that is an object with the attributes to match against if it is present. Then call the function as `primitivesShareAttributeAccessor(primitive, comparePrimitive, {POSITION: true, NORMAL: true})`